handlers/load_test_handler.go

```diff
@@ -36,35 +36,31 @@ func (h *Handler) LoadTestUsingSMPHandler(w http.ResponseWriter, req *http.Reque
 	}()
 	body, err := ioutil.ReadAll(req.Body)
 	if err != nil {
-		msg := "unable to read request body"
-		err = errors.Wrapf(err, msg)
-		logrus.Error(err)
-		http.Error(w, msg, http.StatusInternalServerError)
-		return
+		h.log.Error(ErrRequestBody(err) 
+
+        http.Error(w, ErrRequestBody(err),Error(), http.StatusInternalServerError)
+        return
 	}
 	jsonBody, err := yamlj.YAMLToJSON(body)
 	if err != nil {
-		msg := "unable to convert YAML to JSON"
-		err = errors.Wrapf(err, msg)
-		logrus.Error(err)
-		http.Error(w, msg, http.StatusInternalServerError)
-		return
+		h.log.Error(ErrConversion(err))
+        http.Error(w, ErrConversion(err).Error(), http.StatusInternalServerError)
+        return
 	}
 	perfTest := &SMP.PerformanceTestConfig{}
 	if err := protojson.Unmarshal(jsonBody, perfTest); err != nil {
-		msg := "unable to parse the provided input"
-		err = errors.Wrapf(err, msg)
-		logrus.Error(err)
-		http.Error(w, msg, http.StatusBadRequest)
-		return
+		obj :="provided input"
+        h.log.Error(ErrParseBool(err,obj))
```

Comment:
Whenever there is a "," try to give a space after that it makes the code look tidy.